test/index.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+/* eslint-env mocha */
+/* eslint no-proto: 0 */
+var assert = require('assert')
+var setPrototypeOf = require('..')
+
+function protoTest (obj, proto) {
+  if (Object.getPrototypeOf) {
+    assert.strictEqual(Object.getPrototypeOf(obj), proto)
+  } else if ({ __proto__: [] } instanceof Array) {
+    assert.strictEqual(obj.__proto__, proto)
+  } else {
+    for (var prop in obj) {
+      if (obj.hasOwnProperty(prop)) {
+        assert.strictEqual(obj[prop], proto[prop])
+      }
+    }
+  }
+}
+
+describe('setProtoOf(obj, proto)', function () {
+  it('should merge objects', function () {
+    var obj = { a: 1, b: 2 }
+    var proto = { c: 3, d: 4 }
```

Comment:
best to make proto = { b: 3, c: 4 } to test the precedence of obj field over proto field.

Reply:
I don't think that is necessary.  If the prototype chain is correctly done, then changing this test would only be testing the vm's implementation, which is a waste of effort.  And the test does test that the prototype chain is correct.